docs/getting-started.md

```diff
@@ -34,63 +54,71 @@ scalar.dl.client.cert_path=/path/to/foo.pem
 scalar.dl.client.private_key_path=/path/to/foo-key.pem
 ```
 
+A sample `client.properties` file is available at the `conf` directory, so let's copy it to the current directory.
+
+```shell
+cp conf/client.properties .
+```
+
+Please update the values of the copied `client.properties` file depending on your environment.
+
 ## Register the certificate
 
-Next, let's register your certificate in the Scalar DL network.
-The registered certificate will allow you to register and execute contracts, and will also be used for tamper detection of the data stored in the network.
+Next, let's register your certificate to Scalar DL Ledger.
+The registered certificate will allow you to register and execute contracts and will also be used for Byzantine fault detection of the data stored in Scalar DL.
 
 This time, let's use a simple tool to register your certificate as follows.
 
+```shell
+client/bin/register-cert --properties client.properties
```

Comment:
`client/bin/xxx` doesn't exist yet.
I will update the repo in another PR to have the tools of the corresponding SDK. (At the moment, we only upload a zip file on the release page)